pom.xml

```diff
@@ -906,7 +906,7 @@
                     <limit>
                       <counter>BRANCH</counter>
                       <value>COVEREDRATIO</value>
-                      <minimum>0.73</minimum>
+                      <minimum>0.72</minimum>
```

Comment:
nit: looks like the commit message references the wrong file (Lexer instead of Recognizer)

Reply:
Good catch. Fixed.